StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -342,7 +351,13 @@ class RowButton: UIView {
             return
         }
         // Don't do this if we *are* the tallest variant; otherwise we'll infinite loop!
-        guard sublabel.text?.isEmpty ?? true else {
+        let isSublabelTextEmpty = {
+            if let content {
+                return !content.hasSubtext
+            }
```

Comment:
Why is this needed?

Reply:
In the current state of the refactor it's not required as we still have the "self.sublabel" available, but in the next PR sublabel will be removed and we'll need to look at the content view as we're doing here. Happy to remove and add this later when we really need it.